### ISSUE TITLE
Installs a keyboard in the OS without user's authorization

### ISSUE THREAD

**Issue by blueglyph** [loiste-interactive/infra-issues#1212]
When the game is launched, it does the following modifications to the OS without asking the user's permission:

- installs an US QWERTY keyboard
- activates the keyboard switch hotkey
- makes the language visible in the notification area

This behaviour is not user-friendly, it had made a real mess in my configuration, and it's not necessary.
I have an AZERTY keyboard, and at some point I couldn't log in because I didn't know I was typing my password with the wrong layout.

**Comment by blueglyph** [comment]
I've not been able to remove this, it must be quite a hack.

**Comment by blueglyph** [comment]
That doesn't seem to get much attention. I take it the game is not looked after anymore, seeing the number of open issues.

**Comment by LordLemonDemon** [comment]
Yeah, game seems to be unsupported anymore, and thats sad cuz i like that game very much and i dont think it deserves to die like that.
